newIDE/app/src/Utils/Rectangle.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+export default class Rectangle {
```

Comment:
Just thinking: should this be called "AxisAlignedRectangle"?

Reply:
I don't know. Most frameworks I used call it Rectangle or even Rect. I don't think people could be misled because a rotated rectangle class would be a bit odd.

Reply:
Let's keep Rectangle then :)